community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/codegen/CodeGenerator.scala

```diff
@@ -82,13 +81,7 @@ object CodeGenerator {
   def n = System.lineSeparator()
 
   private def generateCodeFromInstructions(className: String, instructions: Seq[Instruction]) = {
-    val importLines: Set[String] =
-      instructions.
-        map(_.importedClasses()).
-        reduceOption(_ ++ _).
-        getOrElse(Set.empty)
-
-
+    val importLines = instructions.map(_.allImportedClasses).reduceOption(_ ++ _).getOrElse(Set.empty)
```

Comment:
It was easier to read before, however the lines should start with the `.` not end with it.